docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -0,0 +1,133 @@
+
+<video preload="metadata" style="margin-bottom: 16px;" autoplay muted loop controls>
+  <source src="/static/blog/mui-x-mid-v6-features/timezone.mp4" type="video/mp4">
+</video>
+
+Use adapters to control the original value and set the `timezone` prop to define in which timezone the Date, Time, or DateTime should be displayed.
+
+```tsx
+<TimePicker
+  value={value} // Can be in any timezone of your choice
+  onChange={setValue}
+  timezone="Pacific/Honolulu"
+  label={'Rendered in "Pacific/Honolulu"'}
+/>
+```
+
+Please check the [full instructions](https://mui.com/x/react-date-pickers/timezone/) for more information on using this feature in your application.
+
+### Digital clock
+
+This feature is an alternative to the analog clock on time pickers; the original proposal was mainly focused on mobile devices, whereas the new interface is designed for a desktop experience or better time precision.
+This new addition makes our date pickers even more versatile and user-friendly and is available as the default variant for desktops on both [TimePicker](https://mui.com/x/react-date-pickers/time-picker/) and [DateTimePicker](https://mui.com/x/react-date-pickers/date-time-picker/).
+
+<video preload="metadata" style="margin-bottom: 16px;" autoplay muted loop controls>
+  <source src="/static/blog/mui-x-mid-v6-features/timepicker.mp4" type="video/mp4">
+</video>
+
+Check out all the possibilities in [documentation](https://mui.com/x/react-date-pickers/digital-clock/), and let us know your thoughts!
+
+## Data Grid
+
+### Copy and Paste[<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan)
+
+After setting the prop `experimentalFeatures={{ clipboardPaste: true }}` you can copy and paste data directly from and to your Data Grid.
+
+This new feature is extremely versatile, and you can exchange data from other fields, other grids, and even directly from and to the spreadsheet tool of your choice.
+
+<video preload="metadata" style="margin-bottom: 16px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/mui-x-mid-v6-features/copy_paste.mov" type="video/mp4">
+</video>
+
+This feature is integrated with the editing API, so pasted data can be persisted using the `processRowUpdate` prop to update your data source through your usual editing validation process.
+Furthermore, the callbacks `clipboardPasteStart` and `clipboardPasteEnd` are fired during the clipboard paste operation, which can be useful for extra customizations around the event.
+
+For more details on how to use clipboard copy and paste, check out the [feature's page](https://mui.com/x/react-data-grid/clipboard/).
+
+### Filter on column headers[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
+
+This feature allows users to quickly filter the data without needing any additional menu and can be activated using the `unstable_headerFilters` prop.
```

Comment:
Run-on sentence - the two clauses are unconnected. Rephrasing second sentence for active voice.

```suggestion
Filtering on column headers gives users the ability to quickly filter data without any additional menu.
Use the the `unstable_headerFilters` prop to activate this feature.
```